specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -0,0 +1,7956 @@
+        },
+        "readLocations": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains of the read locations enabled for the Cosmos DB account.",
+          "items": {
+            "$ref": "#/definitions/Location"
+          }
+        },
+        "locations": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains all of the locations enabled for the Cosmos DB account.",
+          "items": {
+            "$ref": "#/definitions/Location"
+          }
+        },
+        "failoverPolicies": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains the regions ordered by their failover priorities.",
+          "items": {
+            "$ref": "#/definitions/FailoverPolicy"
+          }
+        },
+        "virtualNetworkRules": {
+          "type": "array",
+          "description": "List of Virtual Network ACL rules configured for the Cosmos DB account.",
+          "items": {
+            "$ref": "#/definitions/VirtualNetworkRule"
+          }
+        },
+        "privateEndpointConnections": {
+          "type": "array",
+          "readOnly": true,
+          "description": "List of Private Endpoint Connections configured for the Cosmos DB account.",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          }
+        },
+        "enableMultipleWriteLocations": {
+          "description": "Enables the account to write in multiple locations",
+          "type": "boolean"
+        },
+        "enableCassandraConnector": {
+          "description": "Enables the cassandra connector on the Cosmos DB C* account",
+          "type": "boolean"
+        },
+        "connectorOffer": {
+          "description": "The cassandra connector offer type for the Cosmos DB database C* account.",
+          "$ref": "#/definitions/ConnectorOffer"
+        },
+        "disableKeyBasedMetadataWriteAccess": {
+          "description": "Disable write operations on metadata resources (databases, containers, throughput) via account keys",
+          "type": "boolean"
+        },
+        "keyVaultKeyUri": {
+          "description": "The URI of the key vault",
+          "type": "string"
+        },
+        "publicNetworkAccess": {
+          "description": "Whether requests from Public Network are allowed",
+          "$ref": "#/definitions/PublicNetworkAccess"
+        },
+        "enableFreeTier": {
+          "description": "Flag to indicate whether Free Tier is enabled.",
+          "type": "boolean"
+        },
+        "apiProperties": {
+          "description": "API specific properties.",
+          "type": "object",
+          "$ref": "#/definitions/ApiProperties"
+        },
+        "enableAnalyticalStorage": {
+          "description": "Flag to indicate whether to enable storage analytics.",
+          "type": "boolean"
+        },
+        "instanceId": {
```

Comment:
please mark as readonly for completeness

Reply:
Made it readonly now